uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/absence-tab/app.js

```diff
@@ -11,17 +11,21 @@ define([
   'leave-absences/absence-tab/components/absence-tab-entitlements',
   'leave-absences/absence-tab/components/absence-tab-work-patterns',
   'leave-absences/absence-tab/components/contract-entitlements',
+  'leave-absences/absence-tab/components/annual-entitlements',
   'leave-absences/shared/components/staff-leave-calendar',
   'leave-absences/shared/models/calendar-model',
   'leave-absences/shared/models/leave-request-model',
   'leave-absences/shared/modules/shared-settings',
-  'leave-absences/shared/models/absence-type-model'
+  'leave-absences/shared/models/absence-type-model',
+  'leave-absences/shared/models/entitlement-model',
+  'leave-absences/shared/modules/shared-settings'
 ], function (angular) {
   angular.module('absence-tab', [
     'ngResource',
     'ui.bootstrap',
     'common.angularDate',
     'common.directives',
+    'common.mocks',
```

Comment:
@deb1990 because the app requires [`Contact`](https://github.com/civicrm/civihr/blob/la-milestone-3/org.civicrm.reqangular/src/common/models/contact.js), and `Contact` requires [`Group`](https://github.com/civicrm/civihr/blob/la-milestone-3/org.civicrm.reqangular/src/common/models/group.js), and `Group` requires [`api.group.mock`](https://github.com/civicrm/civihr/blob/la-milestone-3/org.civicrm.reqangular/test/mocks/services/api/group-mock.js) and [`api.group-contact.mock`](https://github.com/civicrm/civihr/blob/la-milestone-3/org.civicrm.reqangular/test/mocks/services/api/group-contact-mock.js). The two mocks are registered under `common.mocks`, so if we don't include that module as the app dependency, the chain breaks.

The two mocks were part of some work in progress for the Appraisals extension that I never got the chance to finish (here is the [PR](https://github.com/civicrm/civihr/pull/868)). If you remember you had the same problem with the [`Contact`](https://github.com/civicrm/civihr/blob/staging/org.civicrm.reqangular/src/common/models/contact.js#L9) model that you had to fix (linked is the one in 1.6 that sill relies on the mocked data)

I'd say to leave it as it is, fixing it would bloat too much the scope of the PR

@igorpavlov please leave an inline comment near `'common.mocks'` to explain why you need it (you can even link to this comment if you want)